components/infobox/wikis/rocketleague/infobox_league_custom.lua

```diff
@@ -163,6 +166,12 @@ function CustomLeague:defineCustomPageVariables(args)
 	Variables.varDefine('tournament_participant_number', 0)
 	Variables.varDefine('tournament_participants', '(')
 	Variables.varDefine('tournament_teamplayers', args.mode == _MODE_2v2 and 2 or 3)
+	Variables.varDefine('showh2h',
+		Logic.emptyOr(
+			args.showh2h,
+			tostring(tonumber(args.liquipediatier or _H2H_TIER_THRESHOLD) < _H2H_TIER_THRESHOLD)
```

Comment:
```suggestion
			tostring(tonumber(args.liquipediatier) or _H2H_TIER_THRESHOLD < _H2H_TIER_THRESHOLD)
```
Otherwise a non-numeric `args.liquipediatier` will cause errors